Templates/MicrosoftOnlyDefaults.toml

```diff
@@ -0,0 +1,64 @@
+# @file
+# Template Name: New/Secure – Microsoft Only
+# Description:
+#  This template is for systems that need to support only Microsoft certificates.
```

Comment:
Probably should call out signed by 2023 cert.  
I would suggest we check to see if we can identify the version numbers of Windows